web/src/info_overlay.js

```diff
@@ -145,8 +145,7 @@ def zulip():
     },
     {
         markdown: "<time:2023-05-28T13:30:00+05:30>",
-        output_html:
-            '<p><time datetime="2023-05-28T08:00:00Z"><i class="fa fa-clock-o"></i>Sun, May 28, 2023, 1:30 PM</time></p>',
+        output_html: '<time datetime="2023-05-28T13:30:00+05:30"></time>',
```

Comment:
I'm pretty sure this is also intended to match the actual HTML that we generate if you do a Markdown time widget in Zulip. I'm not entirely sure if there's any impact to your change here but it is somewhat concerning.

Reply:
Reverted HTML changes and just changed the time from 1:30PM to 2PM.